NumberBaseball/NumberBaseball.c

```diff
@@ -23,81 +33,85 @@ void Init(int* number1, int* number2, int* number3)
 			continue;
 		}
 
-		*number1 = test1;
-		*number2 = test2;
-		*number3 = test3;
+		computer->guess1 = test1;
+		computer->guess2 = test2;
+		computer->guess3 = test3;
 		break;
 	}
 }
 
-void Update(int* number1, int* number2, int* number3, int* guess1, int* guess2, int* guess3, int* entry)
+void Update(guess* player, int* entry)
 {
 	while (*entry == 1)
 	{
-		char line[LINESIZE];
-		int numbersEntered = 0;
-		*guess1 = -1;
-		*guess2 = -1;
-		*guess3 = -1;
-
-		if (!fgets(line, LINESIZE, stdin))
+		char line[LINE_SIZE];
+		size_t test1 = -1;
```

Comment:
There's no point to have these variables(test1, test2, test3, guess1, guess2, guess3).
Just use player structure pointer